Tests/RxCocoaTests/NSTextView+RxTests.swift

```diff
@@ -12,7 +12,7 @@ import AppKit
 import XCTest
 
 final class NSTextViewTests: RxTest {
-    static let timeout: TimeInterval = 0.5
+    static let timeout: TimeInterval = 2
```

Comment:
why was this change needed?

Reply:
Seems the time the control takes to focus is longer when I ran the tests on Xcode 11, so i increased the timeout.